project-base/config/routes/shopsys_front.yml

```diff
@@ -165,3 +165,10 @@ front_login_window_form:
     defaults:
         _controller: App\Controller\Front\LoginController:windowFormAction
     methods: [POST]
+
+front_download_uploaded_file:
+    path: /file/{uploadedFileId}/{uploadedFileName}
```

Comment:
I think we are not sanitizing diacritic in `uploadFileNam` do not you thing it would be nice to have?

Personally I hate when I download a file and is saved as `U%C5%BEivatelsk%C3%A1%20p%C5%99%C3%ADru%C4%8Dka` instead of `uzivatelska-prirucka`

Reply:
In this case you will download `Uživatelská příručka.pdf` :-)

Reply:
That is awesome. I gues it is `DownloadFileResponse`'s job?

Reply:
Yes exactly